yaks/lib/yaks/format/json_api.rb

```diff
@@ -38,7 +38,7 @@ def serialize_resource(resource)
       def serialize_links(subresources)
         subresources.each_with_object({}) do |resource, hsh|
           next if resource.null_resource?
-          hsh[resource.type] = serialize_link(resource)
+          hsh[resource.rels.first] = serialize_link(resource)
```

Comment:
@plexus I'm not sure if I've completely understood what rels is for. Should I perhaps be tagging rels e.g.

````ruby
['parent_association_name:preferred_works']
```

The idea being that the association name will not always match up with the resource type, so can't rely on resource.type.

I'll dig into the how the mappers work a bit more to make sure I've got this.